### ISSUE TITLE
Fix Invenio/Pride package logic

### ISSUE THREAD

**Issue by kMutagene** [nfdi4plants/arc-validate-package-registry#46]
There are several issues regarding regex/metadata section key terms

It was discussed with @omaus to fix via separate PR instead, as https://github.com/nfdi4plants/arc-validate-package-registry/pull/36 was a crucial unblock for other additions. Note that everything published so far will be wiped before public release anyways as of #44
